script.sh

```diff
@@ -55,7 +55,7 @@ echo '::group:: Running shellcheck ...'
 if [ "${INPUT_REPORTER}" = 'github-pr-review' ]; then
   # erroformat: https://git.io/JeGMU
   # shellcheck disable=SC2086
-  shellcheck -f json  ${INPUT_SHELLCHECK_FLAGS:-'--external-sources'} ${FILES} \
+  shellcheck -f json  ${INPUT_SHELLCHECK_FLAGS:-'--external-sources'} "${FILES}" \
```

Comment:
This does better in that it seems to list out the sh scripts now with spaces, but still is failing:
<img width="771" alt="image" src="https://user-images.githubusercontent.com/9205719/195470048-1e149cc6-0027-4f4d-8c4c-1cba3bdabc31.png">

Reply:
https://github.com/folderol4/shellcheck-test/actions/runs/3238736808/jobs/5307275250